src/components/map/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useRef, useEffect, useState } from 'react';
+import mapboxgl from 'mapbox-gl';
+import style from './map.module.css';
+import 'mapbox-gl/dist/mapbox-gl.css';
+
+export type Position = {
+  lat: number;
+  lng: number;
+};
+
+export type MapProps = {
+  initialPosition?: Position;
+};
+
+export function Map({ initialPosition = defaultPosition }: MapProps) {
+  const mapContainer = useRef<HTMLDivElement>(null);
+  const map = useRef<mapboxgl.Map>();
+  const [position, setPosition] = useState<Position>(initialPosition);
+
+  useEffect(() => {
+    if (!mapContainer.current) return;
+    map.current = new mapboxgl.Map({
+      container: mapContainer.current,
+      accessToken: process.env.NEXT_PUBLIC_MAPBOX_API_TOKEN,
+      style: process.env.NEXT_PUBLIC_MAPBOX_STOP_PLACES_STYLE_URL,
```

Comment:
What happens if these aren't added? Should we show some error? build error?

Reply:
I agree, we should show an error, and it's probably best to do so at build time. Since we are building one image for all organizations I think the best way to do so is to add them to GitHub secrets with the org name suffixed. This way we can pass them as build arguments to docker build. It would require us to update the environment variables in two places though. What do you think? It could look something like this in the Dockerfile: 

```Dockerfile
#Dockerfile
...
ARG ATB_NEXT_PUBLIC_MAPBOX_API_TOKEN
ARG ATB_NEXT_PUBLIC_MAPBOX_STOP_PLACES_STYLE_URL
ARG NFK_NEXT_PUBLIC_MAPBOX_API_TOKEN
ARG NFK_NEXT_PUBLIC_MAPBOX_STOP_PLACES_STYLE_URL
...
# Build AtB
ENV NEXT_PUBLIC_MAPBOX_API_TOKEN=$ATB_NEXT_PUBLIC_MAPBOX_API_TOKEN
ENV NEXT_PUBLIC_MAPBOX_STOP_PLACES_STYLE_URL=$ATB_NEXT_PUBLIC_MAPBOX_STOP_PLACES_STYLE_URL

yarn setup atb
yarn build
...

# Build NFK
ENV NEXT_PUBLIC_MAPBOX_API_TOKEN=$NFK_NEXT_PUBLIC_MAPBOX_API_TOKEN
ENV NEXT_PUBLIC_MAPBOX_STOP_PLACES_STYLE_URL=$NFK_NEXT_PUBLIC_MAPBOX_STOP_PLACES_STYLE_URL

yarn setup atb
yarn build
...

```

Reply:
Maybe it's possible to inject all environments from Github secrets into a separate file that can be included 🤔 As to prevent us from having to add all variables explicitly.

Reply:
But I think your solution is a start and we can work on a more "elegant" solution later.